pip_audit/_util.py

```diff
@@ -3,11 +3,36 @@
 """
 
 import sys
+import tempfile
+import os
 from typing import NoReturn  # pragma: no cover
 
 from packaging.version import Version
 
 
+class CustomNamedTemporaryFile:
```

Comment:
We control the `pip_audit` namespace, so I don't think we need the `Custom` namespace:

```suggestion
class NamedTemporaryFile:
```

and then just `from pip_audit._util import NamedTemporaryFile` wherever we need it 🙂